src/tests/rollbar-react/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "../../index.js",
+  "types": "../../../index.d.ts"
+}
```

Comment:
For Typescript to consume the package under test it needs both an entry point and the type declarations. If they were in the same directory, it could find the type declarations automatically. They are in different directories though, so we use a package.json specific to the test environment, and Typescript knows how to consume this correctly.